api/server/volume.go

```diff
@@ -165,7 +166,13 @@ func (vd *volAPI) create(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	id, err := d.Create(dcReq.Locator, dcReq.Source, dcReq.Spec)
+	spec, err := sdkVol.GetEnforcedVolSpecs(dcReq.Locator, dcReq.Source, dcReq.Spec)
```

Comment:
I thought this was going to be in the driver. If it is here, that is find. Just add to the Docker API and CSI API also.